code/eyetracking/eyetrackingrun.py

```diff
@@ -325,6 +326,53 @@ def __init__(
                 for msg_timestamp, msg in messages[["trialid_time", "trialid"]].values
             ]
 
+        # Parse calibration metadata
```

Comment:
why the reordering? and is it only a reordering or there are minor changes here and there?

Reply:
The calibration message is extracted during the parsing of the `messages` dataframe and it does not depend on the parsing of the `recording` dataframe, as such it is a more logical flow to explicit it after parsing messages dataframe rather than after `recording` dataframe.

Reply:
maybe it's not necessary to reorder in the code but in the notebook it is IMHO.